wled00/FX.cpp

```diff
@@ -388,6 +388,86 @@ uint16_t WS2812FX::mode_rainbow_cycle(void) {
   return FRAMETIME;
 }
 
+/*
+* moves a dot in primary color from left to right
+* dot width and speed is variable
+* dot fade in/out smooth
+* act as multi line display with a width of 36 columns (maybe later variable!?)
+*
+* SEGMENT.intensity = width of the dot
+* SEGMENT.speed = motion speed
+* ........................xXXx..->....
+* ........................xXXx..->....
+* ........................xXXx..->....
+*/
+
+uint16_t WS2812FX::mode_lighthouse2()
+{    
+  int rowLength = 36;
```

Comment:
Why 36? Not everyone wants a row length of 36.

Reply:
changed it to be configurable with parameter 2, now the dotwidth is fixed for now (but will be variable again with next version)

Reply:
Taking 3rd parameter will be available in 2D version of WLED sometime after summer.